src/main/java/org/codefx/libfx/control/webview/WebViewHyperlinkListener.java

```diff
@@ -2,19 +2,20 @@
 
 import java.util.function.Function;
 
-import javafx.scene.web.WebEngine;
-import javafx.scene.web.WebView;
-
 import javax.swing.event.HyperlinkEvent;
 import javax.swing.event.HyperlinkListener;
 
+import javafx.scene.web.WebEngine;
+import javafx.scene.web.WebView;
+
```

Comment:
Can you return this to the original import order (not that it is better in any way, I just want to reduce the chance of "no-op changes").